README.md

```diff
@@ -6,13 +6,24 @@ sentry internal pypi
 this repository contains the tools to import and/or build packages from public pypi for the
 platforms and achitectures required for sentry development.
 
+## setup
+
+```
+# install tox (if not already installed)
+brew install tox
+
+# create your development venv
+tox devenv venv
+```
+
 ## adding packages
 
 packages are configured in the `packages.ini` file.
 
 the easiest way to add a package and its dependencies is to use:
 
 ```bash
+source venv/bin/activate
```

Comment:
I guess my overall point is we shouldn't duplicate tox's docs or virtualenv's docs -- at _most_ we should link to those but it should be assumed given the directory structure

Reply:
I'm mainly used to tox being used as a test runner against a matrix of dependencies (or building in this case) but not as a way to run tasks within a specific env (like the `add_pkg` cmd) - this is the first time I've come across `tox devenv` (neat!).

I'd be curious if other engineers would infer the tox workflow based on the repo layout or setup something with whatever they're most familiar with like I did. I think some guidance towards a standard approach would be good, but I'll leave that to your discretion.

Another option would be automating this with direnv like we do on some other repos - open to a PR for that?

Reply:
I'm personally not a fan of direnv -- it's too much action at a distance and has [some safety flaws](https://github.com/direnv/direnv/issues/445)